### DISCUSSION TITLE
Getting DisplayObject's x, y, width, height by getLocalBounds()

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7708] [Q&A]
I'm using DisplayObject.getLocalBounds()
I'll invoke getLocalBounds() so often (by mouseMove)
It works well.
But is it ok for performance aspect?
I read this, so i'm a bit worred. https://github.com/pixijs/pixijs/issues/6355

Another question is, why is there no DisplayObject.width or height property  instead using  getLocalBounds()?
Thanks!

**Comment by ivanpopelyshev** [comment]
use `getBounds()` when you can.

> why is there no DisplayObject.width or height property instead using getLocalBounds()?

its the same, just multiplied by scale. if you change width/height then you change scale. It sometimes works :)

**Reply by humbleCoder1990** [reply]
getBounds() has better performance than getLocalBounds()?

**Reply by humbleCoder1990** [reply]
Another question..
How about using 'InteractionData.getLocalPosition()?' 
Is it expenssive?

**Reply by ivanpopelyshev** [reply]
No, its calling `toLocal` whjich is updating transform sometimes and reverses matrix, e.t.c. 

You have downloaded pixi sources and opened in separate IDE window to view classes by shortcut, right?
